### DISCUSSION TITLE
"You don't have permission to use this app"

### DISCUSSION THREAD

**Discussion by xpufx** [Budibase/budibase#4467] [Help]
![image](https://user-images.githubusercontent.com/25581961/153773975-5318c059-7ce9-4ea0-80f1-6fdf4014c252.png)

This is the message I get when I click the play button on the design page or 'View Published app" in My Apps. The user who's creating the app and the one viewing it is the same. It also has ADMIN rights. It's the same session too. I can keep editing but not launch the app.

When I go to the portal at /builder/apps I can see this app listed for this user as well. But when I click the app I get the same error.

**Comment by xpufx** [comment]
This sort of kind of started working when I added a "SCREEN". I am just evaluating and don't know why this fixes it.

**Reply by jaspreet1313** [reply]
When i make app and publish it , it shows same error . I got the solution .
In the users section of sidebar , remove your app from no access option. Make it basic. I hope it will work.

**Reply by mjashanks** [reply]
@xpufx I see - makes sense that this would happen if there are no screens added. I think it's falling back to that screen thinking "the users does not have access to any screens!"
